ci/github/unit-testing/webserver.bash

```diff
@@ -9,10 +9,28 @@ install() {
     pip list -v
 }
 
-test() {
+test_isolated() {
     pytest --cov=simcore_service_webserver --durations=10 --cov-append \
           --color=yes --cov-report=term-missing --cov-report=xml --cov-config=.coveragerc \
-          -v -m "not travis" services/web/server/tests/unit
+          -v -m "not travis" services/web/server/tests/unit/isolated
+}
+
+test_with_db_slow() {
+    pytest --cov=simcore_service_webserver --durations=10 --cov-append \
+          --color=yes --cov-report=term-missing --cov-report=xml --cov-config=.coveragerc \
+          -v -m "not travis" services/web/server/tests/unit/with_dbs/slow
+}
+
+test_with_db_medium() {
+    pytest --cov=simcore_service_webserver --durations=10 --cov-append \
+          --color=yes --cov-report=term-missing --cov-report=xml --cov-config=.coveragerc \
+          -v -m "not travis" services/web/server/tests/unit/with_dbs/medium
+}
+
+test_with_db_fast() {
+    pytest --cov=simcore_service_webserver --durations=10 --cov-append \
+          --color=yes --cov-report=term-missing --cov-report=xml --cov-config=.coveragerc \
+          -v -m "not travis" services/web/server/tests/unit/with_dbs/fast
```

Comment:
> Could you please explain more what is the meaning of the following:
> 
> * isolated
> * db slow
> * db medium
> * db fast
> 
> It would also be great to see a compressed version in the CI.


Yes, I guess we could put this document in a more visible place

https://github.com/ITISFoundation/osparc-simcore/blob/e8d960b9549ad591309dfd24f7c31c2d69fb96fd/ci/travis/unit-testing/README.md

Reply:
isolated = these tests are (IMO) real unit tests, they do not need any dependencies and were already in the root test/unit folder before
db slow = it is currently the slowest test module it takes about 10 minutes (db stand for database-dependency, these tests were already in test/unit/with_dbs folder)
db medium = are tests that take 3-5 minutes, altogether about 10 minutes (db stand for database-dependency, these tests were already in test/unit/with_dbs folder)
db fast = all the others (db stand for database-dependency, these tests were already in test/unit/with_dbs folder)

As the plan is to strip the webserver into small micro-services I did not create now a super fancy classification but merely split the tests in ~equivalent test times. 

I will add a link to this PR in the github workflow.

Reply:
I would put this information directly In on top of the bash functions like a comment.

Reply:
here we go